app/views/quotes/new.html.erb

```diff
@@ -154,6 +272,27 @@
    </div>
 </div>
 
+<!-- REQUEST CALL MODAL -->
+<div class="modal fade" id="requestCallSubmitModal" tabindex="-1" role="dialog" aria-labelledby="requestCallSubmitModalLabel" aria-hidden="true">
+   <div class="modal-dialog modal-dialog-centered" role="document">
+      <div class="modal-content">
+         <div class="modal-header bg-primary">
+            <h5 class="modal-title text-white text-shadow text-center" id="requestCallSubmitModalLabel">Enter your phone and one of our friendly mangers will get in touch with you as soon as possible</h5>
+         </div>
+         <%= simple_form_for(@lead, remote: true) do |form| %>
+            <div class="modal-body">
+               <%= form.input :phone_number %>
+               <%= form.input :first_name %>
+               <%= form.input :last_name %>
+               <div class="text-center">
+                  <%= form.submit "Request a call", class: 'btn btn-dark ml-3' %> Or call 778-SHOVEL-SQUAD
```

Comment:
```suggestion
                  <%= form.submit "Request a call", class: 'btn btn-dark ml-3' %> Or call <a href="tel://778-SHOVEL-SQUAD">778-SHOVEL-SQUAD</a>
```

Reply:
Place holder until they give me the content